src/Resources/contao/forms/FormPassword.php

```diff
@@ -124,9 +124,12 @@ protected function validator($varInput)
 			return '';
 		}
 
-		if (Utf8::strlen($varInput) < \Config::get('minPasswordLength'))
+		// Check password length either from DCA or use Config as fallback (#1087)
+		$intLength = ($this->minlength > 0) ? $this->minlength : \Config::get('minPasswordLength');
```

Comment:
This can be simplified to `$intLength = $this->minlength ?: \Config::get('minPasswordLength');`